.travis.yml

```diff
@@ -8,14 +8,33 @@ dist: xenial
 language: python
 
 cache: pip
-python:
-  - '2.7'
-  - '3.5'
-  - '3.6'
-  - '3.7'
-  - '3.8'
-  - 'pypy2.7-6.0'
-  - 'pypy3.5-6.0'
+matrix:
+  include:
+    - arch: arm64
+      python: 2.7
+    - arch: amd64
+      python: 2.7
+    - arch: arm64
+      python: 3.5
+    - arch: amd64
+      python: 3.5
+    - arch: arm64
+      python: 3.6
+    - arch: amd64
+      python: 3.6
+    - arch: arm64
+      python: 3.7
+    - arch: amd64
+      python: 3.7
+    - arch: arm64
+      python: 3.8
+    - arch: amd64
+      python: 3.8
+    - arch: amd64
+      python: pypy2.7-6.0
+    - arch: amd64
+      python: pypy3.5-6.0
+      # Skipping for Arm64, pypy has aarch64 support - but it isn't released yet.
```

Comment:
I find this very hard to read with 'arm64' and 'amd64' interspersed. Can the architectures be grouped together to make it more readable?

Also, python 2.7 can be omitted for arm64 as it is EOL.